students/minji/users/views.py

```diff
@@ -42,12 +45,20 @@ class LogInView(View):
     def post(self, request):
         try:
             data     = json.loads(request.body)
+
             email    = data['email']
             password = data['password']
 
-            if not User.objects.filter( email = email, password = password).exists():
+            user = User.objects.get(email = email)
+
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')): 
                 return JsonResponse({'message':'INVALID_USER'}, status=401)
-            return JsonResponse({'message':'SUCCESS'}, status=200)
+
+            token = jwt.encode({"user_id":user.id}, settings.SECRET_KEY, settings.ALGORITHM)
+            return JsonResponse({"message":"SUCCESS", "access_token": token}, status=200)
         
+        except User.DoesNotExist:
+            return JsonResponse({"message":"INVALID_USER"}, status=401)
```

Comment:
```suggestion
            return JsonResponse({"message":"INVALID_USER"}, status=400)
```

401(Unauthorized) 응답을 받는 대표적인 경우는 로그인이 되어 있지 않은 상태에서 무언가 요청을 하는 경우입니다. 
예를 들어 어떤 이커머스 사이트에 로그인을 하지 않은 상태로 사용자의 결제 내역과 같은 정보를 달라고 하면 401(Unauthorized)를 반환받는 것을 예로 들 수 있습니다.

status code를 적절하게 수정해주세요!

Reply:
수정....했습니다................😇